### DISCUSSION TITLE
Export conversions to Google Ads

### DISCUSSION THREAD

**Discussion by ChrisLamprecht** [plausible/analytics#1628] [Anything Else]
Is there a way to connect Plausible to Google Ads and avoid the Google Ads cookie for conversion Tracking?

**Comment by metmarkosaric** [comment]
no, that's not possible. you can track conversions in Plausible itself but we don't have a way to send any data to Google

**Reply by ChrisLamprecht** [reply]
Thank you! But I can use the stats api for this or did I get this wrong?
I would love to ditch Google Analytics, but we still need Google Ads for our business and without conversion tracking this makes no sense.
Otherwise I think about using etracker.com, but I like Plausible much more!

**Reply by metmarkosaric** [reply]
you can export your conversion stats via the [CSV](https://plausible.io/docs/export-stats) or the [API](https://plausible.io/docs/stats-api). does google allow you to import data? if yes, then you should be able to import the data you take from Plausible 

you can track ads conversions directly in Plausible without the need for Google ads. we support both UTM tags and conversion tracking so you can follow the journey from a click on a paid ad to a conversion on your site and see which campaigns perform best. then you can tweak your ad campaigns in Google ads accordingly without needing to send them any data

**Reply by ChrisLamprecht** [reply]
I will give it a try! :-)

**Reply by metmarkosaric** [reply]
cool! do update here please if you find a way to import our data to google so others can know it too. thanks!

**Reply by rahlswede** [reply]
Is there an update / solution regarding the return of conversion values? Nearly all ad types are using signals for optimisation.

**Reply by gaa23** [reply]
would like to see this done also

**Reply by Zielgestalt** [reply]
It's possible to upload conversions to Google Ads for 'offline conversion tracking'. So it should be possible: https://support.google.com/google-ads/topic/7280668?hl=en&ref_topic=3119145

**Reply by larsschellhas** [reply]
Hm, that looks interesting! Would certainly love for this to work to use the "Conversion maximising" bidding strategies.

[a comment by an automated account is not shown]

**Comment by JPustkuchen** [comment]
Also see https://www.etracker.com/docs/integration-setup/kampagnen-tracking/google-ads/anleitung-conversion-upload/

eTracker does it :) It's possible!

Here for facebook: https://www.etracker.com/docs/integration-setup/kampagnen-tracking/facebook-instagram-ads/anleitung-conversion-upload/

**Reply by JPustkuchen** [reply]
And this is how Matomo does it: https://matomo.org/faq/reports/how-to-set-up-conversion-import-in-google-ads/

Super important feature for Google Ads users (which is most of us... ;))

**Reply by JPustkuchen** [reply]
And this is how Mixpanel does it: https://docs.mixpanel.com/docs/cohort-sync/integrations/google-ads

**Comment by dhilgarth** [comment]
Is there any news from the maintainers regarding this? @metmarkosaric 
I'm currently paying for Plausible and love it, but as others have said: Google Ads optimize based on the conversions, so this data is needed inside Google Ads.

**Reply by MickL** [reply]
6 months later, same question :) It is possible with Etracker and Matomo to upload the data into Google Ads. Can we have the same for Plausible?

**Comment by Bartholomewcodermaster** [comment]
Google ads is very effective, but Conversion Value is an important issue. If anyone is looking for a good post on this topic, I recommend this post: https://gamerseo.com/blog/what-is-conversion-value-google-ads-2023-guide/

**Comment by markwilson860** [comment]
Export conversions to Google Ads is a critical strategy for optimising digital marketing campaigns. By integrating conversion data from different sources such as websites, mobile applications or offline sales into Google Ads, marketers can better understand which keywords, ads and campaigns lead valuable actions. This data-based approach allows for more targeted targeting and budgetary allocation, increasing return on investment (ROI). The process usually involves establishing [Google Ads conversion](https://clicksyou.com/service/conversion-optimisation/) tracking or importing conversion data from other platforms using tools like Google Analytics or third-party CRM systems. Once imported, this data helps to refine supply strategies, adapt advertisers and improve overall campaign performance. Therefore, companies can achieve more efficient marketing results by ensuring that their advertising efforts are directly aligned with their most important indicators.

**Comment by EscaleCommunication** [comment]
This is becoming critical, as customers want to maximize their google ads campaigns. 

The thread has been open for more than a year and a half without any movement, even though it's a vital need for freelancers and agencies. 

Plausible is a superb tool and many of my clients are happy with its simplicity and operation, but it's got them so tied up that we're thinking of going back (Google) or moving on to other options. 

Would it be possible to have visibility on a roadmap for this precise functionality ? I don't really want to leave plausible but client are pushing for better ads campaign (max perf, conversion based...) @metmarkosaric

**Reply by JPustkuchen** [reply]
I can confirm that, thanks!

**Reply by MickL** [reply]
I agree! It is possible with Etracker and Matomo to upload the data into Google Ads. Can we have the same for Plausible? @metmarkosaric

**Comment by danielsiegl** [comment]
Any news? This is really a must have capability.

**Comment by StacySiz** [comment]
Hey everyone! 👋
Just wanted to share that if you're interested in seeing the "Export Conversion to Google Ads" feature implemented, you can vote for it here: 
https://plausible.nolt.io/210

While the Plausible team might not actively monitor this thread, they do prioritize features based on community votes. Currently, this request only has 30 upvotes – if you think this would be useful, please consider adding your vote to help get it on their radar!

Thanks!

**Comment by larsschellhas** [comment]
Actually, they posted an article end of last year about why it doesn't work with Google Ads (currently).

Main issue being that they strip all unique identifiers of which at least one would be required to be provided to Google in a CSV upload.

Of course they could change that behaviour, but it would impact the privacy-friendliness and probably make consenslt necessary. 

https://plausible.io/blog/google-ads-tracking
